mesa_frames/types.py

```diff
@@ -1,20 +1,22 @@
-from typing import Collection, Hashable, Literal
+from collections.abc import Collection, Hashable
+from typing import Literal
+
+import pandas as pd
```

Comment:
The problem with these imports is that user will have to install both pandas and Polars.

Reply:
You're right, but I don't have a clear idea on how to type hint `DataFrame`, `Series `ecc. properly otherwise..

Reply:
I assume putting the imports within the `if TYPE_CHECKING:` doesn't work on this file? If it doesn't work, then it's fine, package storage is not that big of an issue nowadays anyway.

Reply:
Unfortunately it doesn't because type aliases are defined as variables importable from other modules. Maybe there's a better way to solve this but I need to think about it